include/deal.II/base/mpi.h

```diff
@@ -267,19 +267,6 @@ namespace Utilities
     class MPI_InitFinalize
     {
     public:
-
-      /**
-       * Constructor. Takes the arguments from the command line (in case of
-       * MPI, the number of processes is specified there), and sets up a
-       * respective communicator by calling <tt>MPI_Init()</tt>. This
-       * constructor can only be called once in a program, since MPI cannot be
-       * initialized twice.
-       *
-       * This constructor sets max_num_threads to 1 (see other constructor).
-       */
-      MPI_InitFinalize (int    &argc,
-                        char ** &argv) /*DEAL_II_DEPRECATED*/;
-
```

Comment:
invalid_unsigned_int would also be okay as a default. The only reason to specify 1 is if you are using PETSc. But we have an exception in place that triggers on startup if you use PETSc and don't specify 1.